client/src/app/store/store.js

```diff
@@ -6,16 +6,14 @@ import {composeWithDevTools}from 'redux-devtools-extension'
 
 
 
-import reducer from '../reducers/rootReducers'
+import rootReducer from '../reducers/rootReducers'
 
 
-
-
-const  store = createStore(
-    rootreducer, composeWithDevTools(applyMiddleware(thunk))
+export const store = createStore(
```

Comment:
You'll probably want to pass it the initial state

Reply:
Done